### ISSUE TITLE
My wallet no syncing

### ISSUE THREAD

**Issue by illuSionRiDer** [PaycoinFoundation/paycoin#291]
Don't know what happens please help me!!
![2015-10-23 19 08 34](https://cloud.githubusercontent.com/assets/15263885/10692643/32401aa0-79bd-11e5-98d8-ec2af9703178.png)
![2015-10-23 19 28 11](https://cloud.githubusercontent.com/assets/15263885/10692644/37fb2b24-79bd-11e5-8551-36a8a04ccc75.png)

**Comment by illuSionRiDer** [comment]
Just download and installed it but it seems the wallet never sync, and why with zero active connection? somebody pls help me!!

**Comment by IngCr3at1on** [comment]
This was happening in some instances even on the old wallets (both 0.3.1.0 and 0.3.1.1), off the top of my head I'm not sure of the actual cause but I've seen it happen on many wallet versions.

**Comment by illuSionRiDer** [comment]
Now it syncs, thanks my all friends!!
![2015-10-23 21 54 57](https://cloud.githubusercontent.com/assets/15263885/10695985/f51ecd7e-79d0-11e5-8a72-7f29a5c96dd3.png)

**Comment by illuSionRiDer** [comment]
Do you have a bootstrap for that, so I think it could go real faster?

**Comment by RdeWilde** [comment]
https://www.dropbox.com/s/3rpfrodz9a1l94m/BlockChain.zip?dl=0

**Comment by IngCr3at1on** [comment]
We were trying to avoid bootstrapping until after the fork just to make sure that the chain being shared is most definitely the same.

**Comment by illuSionRiDer** [comment]
Thanks so much friend I'll try it out 😁😁

**Comment by illuSionRiDer** [comment]
What do you mean? I better not to use the bootstrapping?

**Comment by IngCr3at1on** [comment]
I mean @RdeWilde seems pretty convinced it's a valid bootstrap I just know that's why we (the dev team) weren't sharing one at this time. If anything I'd say use at your own risk.

**Comment by illuSionRiDer** [comment]
Got it Sir!! Thanks a lots!!

**Comment by IngCr3at1on** [comment]
@illuSionRiDer seems he just checked again and he's not so sure? I'm multitasking a bit, I'd recommend waiting on a normal sync iirc

**Comment by illuSionRiDer** [comment]
Okay I'll wait!!

**Comment by mwwhited** [comment]
I'm having this same issue...  any ETA?  I thought this was a bug that was supposed to be fixed in 3.2.0?

**Comment by illuSionRiDer** [comment]
I got it done well now with using the bootstrap, but the problem is my 9 xpy coins never confirm, and don't know what to do!!
![2015-10-28 11 55 23](https://cloud.githubusercontent.com/assets/15263885/10780001/f9e3b1b0-7d6a-11e5-840f-d47fb61cd5fd.png)

**Comment by mitchellcash** [comment]
@mwwhited I think possibly a separate syncing issue is going on. This one in particular was solved. Please see #294 for the new issue. The quickest solution I believe is bootstrapping but we do know the fix but we just need to finalise testing and have it merged into the codebase.

**Comment by illuSionRiDer** [comment]
@mitchellcash  and what about mine, how can I get my 9 coins back to the wallet please++

**Comment by IngCr3at1on** [comment]
Synching issues are resolved if you build from master otherwise we'll be pushing a release with this fix. Closing issue.
